### PR TITLE
Add gnosis safe airdrop address data

### PR DESCRIPTION
**Pull request by springzh**
Brief comments on the purpose of your changes:

Added value based data table for gnosis safe airdrop data. Source file downloaded from https://raw.githubusercontent.com/safe-global/safe-user-allocation-reports/main/safe_user_allocations_reworked.csv 

*For Dune Engine V2*
I've checked that:
General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by soispoke — CHANGES_REQUESTED** [review]
Thanks for your submission @springzh ! 

Could you please change reformat this query so the model is addresses_ethereum.safe_airdrop so we can standardize airdrops across different chains ? 
https://github.com/duneanalytics/spellbook/pull/1529?no-redirect=1

File: macros/alter_table_properties.sql [hunk 1]
```diff
@@ -415,6 +415,15 @@ ALTER TABLE airdrop_optimism.addresses_1 SET TBLPROPERTIES('dune.public'='true',
                                                     'dune.data_explorer.contributors'='["soispoke"]');
 {% endset %}
 
+{% set addresses_ethereum_safe_airdrop %}
+ALTER TABLE addresses_ethereum.safe_airdrop SET TBLPROPERTIES('dune.public'='true',
+                                                    'dune.data_explorer.blockchains'='["ethereum"]',
+                                                    'dune.data_explorer.category'='abstraction',
+                                                    'dune.data_explorer.abstraction.type'='sector',
+                                                    'dune.data_explorer.abstraction.name'='safe_airdrop',
```
Comment (springzh) [inline macros/alter_table_properties.sql:423]: Not sure about this line. What's the usage for the abstraction name? Should it be always set to same as table name?

**Comment by springzh** [comment]
> Thanks for your submission @springzh !
> 
> Could you please change reformat this query so the model is addresses_ethereum.safe_airdrop so we can standardize airdrops across different chains ? #1529

Updated. Please check again. @soispoke

File: macros/alter_table_properties.sql (hunk 1 above)
Comment (soispoke) [inline macros/alter_table_properties.sql:423]: Here, I think the type should be `sector`, and the name `addresses`

**Review by soispoke — CHANGES_REQUESTED** [review]

**Comment by soispoke** [comment]
The model file name should be addresses_ethereum_safe_airdrop.sql instead of safe_airdrop.sql

File: macros/alter_table_properties.sql (hunk 1 above)
Comment (springzh) [reply macros/alter_table_properties.sql]: Updated

**Comment by springzh** [comment]
> The model file name should be addresses_ethereum_safe_airdrop.sql instead of safe_airdrop.sql

@soispoke Renamed. Thank you.

**Review by soispoke — APPROVED** [review]
lgtm !

**Pull request merged**